### DISCUSSION TITLE
Catppuccin theme has a weird window

### DISCUSSION THREAD

**Discussion by dorekhov1** [LazyVim/LazyVim#2661] [Q&A]
I am pretty new to using neovim and I am currently trying to configure lazyvim the way I like it. I am using the catppuccin theme and everything seems fine except for this weird orange window that shows up for explanation of completion suggestions:

![image](https://github.com/LazyVim/LazyVim/assets/27707320/25fe2720-2b2b-4c40-9164-b69ab6bf2490)

Since I am new to all this, I am not sure what this window is called / which plugin it comes from and how I would adjust its colors, so I am not sure where to even start to fix the theme. Any help would greatly appreciated.

**Comment by alivault** [comment]
Sometimes I get issues like this when switching the colorscheme without reloading nvim. Have you tried quitting and reloading nvim?

**Reply by dorekhov1** [reply]
yeah, it is like that all the time

**Comment by dangdd2003** [comment]
I'm using catppuccin too, that orange window is lsp doc as i remember. What is your configuration with this theme ?

**Comment by dorekhov1** [comment]
It turned out this was because of tmux not having true colors enabled properly. Fixing tmux  fixed this orange window
